package.json

```diff
@@ -83,7 +83,6 @@
     "eg:goerli:balances": "ts-node -r dotenv/config examples/scripts/balances.ts dotenv_config_path=./examples/scripts/.env.goerli",
     "eg:goerli:export-commitments": "ts-node -r dotenv/config examples/scripts/exportCommitments.ts dotenv_config_path=./examples/scripts/.env.goerli",
     "eg:goerli:import-commitments": "ts-node -r dotenv/config examples/scripts/importCommitments.ts dotenv_config_path=./examples/scripts/.env.goerli",
-    "eg:web-app": "npm run build && cd examples/webApp/ && npm run serve",
     "eg:react-web-app-start": "npm run build && cd examples/web-app/ && npm start",
```

Comment:
`eg:start-react-app` (?)

Reply:
idk, we can do both, this is a convention suggestion by Pawel. I like them both.